statement.go

```diff
@@ -626,14 +631,19 @@ func clusteringFieldOrSentinel(term interface{}) interface{} {
 			return []byte(ClusteringSentinel)
 		}
 		return v
+	case time.Time:
+		if v.IsZero() {
+			return ClusteringSentinelTimestamp
+		}
+		return v
```

Comment:
Is this what we want? 🤔 

My recollection of this was that Cassandra lets you store an empty `string` in a column of type `timestamp` which keyspaces doesn't. I think that means that the `term` here would be a `string` and we need to understand the column type to determine whether to use `ClusteringSentinelTimestamp` or the `ClusteringSentinel` 🤢 

I could be misremembering though